GOVERNANCE.md

```diff
@@ -0,0 +1,54 @@
+The **WGPU project** is an ecosystem that _enables application authors to write
+portable and performant graphics programs_. It was originally conceived to
+provide an implementation of WebGPU for Firefox as the standard evolved and
+settled into something that could be shipped on all web browsers. WGPU has also
+enjoyed much contribution and use from other projects that require graphics
+programming. We expect that these sorts of users will continue for the lifetime
+of project, and we embrace these contributors' needs and effort as the lifeblood
+of WGPU.
+
+## Mission
+
+The WGPU community seeks to realize the following directives through the
+project: it…
+
+1. …provides libraries for the WebGPU API that…
+    1. …is correct and fully conformant.
+    1. …is portable across all major platforms, that is, …
+        1. …`wgpu-core` enables JavaScript platforms to implement their own
+           proper WebGPU API.
+        1. …`wgpu` provides a WebGPU-style API library for native applications,
+           which allows shipping to all major platforms, including WebGPU's
+           JavaScript API.
+    1. …is performant enough to enable demanding applications.
+1. …serves as a platform of experimentation for:
+    1. …WebGPU standards development.
+    1. …native application authors that wish to experiment with features that
+       are not (yet?) standard.
+
+## Decision-making
+
+The WGPU community's decision-making is influenced by the following
+groups:
+
+* Community leadership:
+    * Connor Fitzgerald (@cwfitzgerald)
+    * Joshua Groves (@groves)
+    * Andreas Reich (@wumpf)
+* Firefox's WebGPU team (@jimblandy, @teoxoy, @ErichDonGubler, and others)
```

Comment:
I will take mild offense on nical's behalf, he definitely belongs on this list :)

Reply:
ofrick urite sorry @nical 😭

This is why I didn't wanna name people, yo.

Will fix soon.

Reply:
Fixed with 598bc0ce9.